### DISCUSSION TITLE
Re-introduce a way to disable `AUTH_REQUIRED`

### DISCUSSION THREAD

**Discussion by ornanovitch** [lukevella/rallly#891] [Ideas]
Hi,

First of all thank you for this brilliant app. It's been a while since I was searching a Doodle alternative for a small non-profit tech collective I'm part of. When a friend pointed me your website, I felt we found the right tool. One of the better feature you seemed to offer for us is the ability to create polls as guest. But then, after starting to prepare a test instance, I realized it was more complicated. As you stated in the V3 release note:

> The `AUTH_REQUIRED` parameter has been removed. As of version 3, users must be logged in to create and manage polls

I was a little disappointed and I was near to abandon this path, but then I found this sentence in your doc, about the rallly.co demo instance:

> This version differs slightly from the self-hosted version in that it allows guest users to create polls without having to create an account

Therefor I must ask you: is there a specific reason not to integrate this feature into the self-hosted app? Is there a way for us to integrate it by building the app ourself? Are you willing to at least explain how does it work?

Anyway, keep this good work, its precious!

Thanks

**Comment by Tuphal** [comment]
Requireing a login for guests is a huge inconvenience...

A way to allow guests for voting (at least) would be great.
Please reintroduce AUTH_REQUIRED

**Reply by lukevella** [reply]
Guests are allowed to vote through the invite links in the self-hosted version. You'd only need to login to create a poll.

**Reply by Tuphal** [reply]
thank you for clarification.
The issue is the link `domain.com/p/PytC77vzxtIf` which is not automatically redirected to `domain.com/invite/xQZ5yuzaJ8vw`. Only when one is logged in it is redirected.

**Reply by lukevella** [reply]
Ah right. That's a bug since the redirect from the older URLs should be public. I would strongly urge to move away from the redirect though as they will deprecated and removed at some point. It's only intended to be there to help transition between versions.

**Comment by lukevella** [comment]
The current behavior seems sensible to me as I don't see the point of running a self-hosted version that is completely open to the public. That's what the managed service is for. I might bring it back in a future version as an opt-in feature but at the moment it would not be trivial to make this work and this is sitting quite low on the priority list.

**Reply by ornanovitch** [reply]
I understand your point, but IMHO as you suggest this should only be an opt-in feature. Regarding the interest of having multiple public instances, I think this is more a matter of contributing to an ethical challenge AKA redecentralise the web. The possibility to have multiple "managed services", with rallly.co as the mothership, is a very wise path (just as https://meet.jit.si/ is coexisting with several Jitsi Meet public instances).

I hope you'll have a chance to consider this again in the future.

**Reply by ornanovitch** [reply]
And thanks again for your work!

**Reply by L041S** [reply]
Hello,
I found this discussion because I’m interested in this feature as well, and I wanted to explain why it could be useful. Personally, I self-host everything I can to keep data on my own server, but I also want my self-hosted services to be as convenient as possible for my friends and family.
For this reason, I’m looking for a way to allow guests to create polls without the friction of creating an account, while still keeping the poll data on my own server. This would let my friends and family create polls without having to use a service that doesn’t belong to us.
